cmd/migrate/main.go

```diff
@@ -0,0 +1,124 @@
+package main
```

Comment:
just thinking of the filename `migrate` we have different types of migrations. Could we be more clear e.g. `db_migrate` or something?